pkg/cmd/cli/cmd/edit.go

```diff
@@ -23,14 +22,27 @@ import (
 	"github.com/openshift/origin/pkg/util/jsonmerge"
 )
 
+type EditOptions struct {
```

Comment:
I agree that we should have a comment preceding the struct definition. By the way, I noticed that we're missing that in other CLI commands as well, including the ones listed by @kargakis:
- [pkg/cmd/cli/cmd/deploy.go#L26](https://github.com/openshift/origin/blob/255307e708defdbdb66fc421cc3d45cce375a209/pkg/cmd/cli/cmd/deploy.go#L26)
- [pkg/cmd/cli/cmd/project.go#L25](https://github.com/openshift/origin/blob/255307e708defdbdb66fc421cc3d45cce375a209/pkg/cmd/cli/cmd/project.go#L25)
- [pkg/cmd/cli/cmd/logout.go#L20](https://github.com/openshift/origin/blob/255307e708defdbdb66fc421cc3d45cce375a209/pkg/cmd/cli/cmd/logout.go#L20)
- [pkg/cmd/cli/cmd/request_project.go#L20](https://github.com/openshift/origin/blob/255307e708defdbdb66fc421cc3d45cce375a209/pkg/cmd/cli/cmd/request_project.go#L20)
- etc...

We might want to do a more massive "add missing godocs".
@jprukner @romanbartl you could add the missing doc line here and later open a new PR to add a similar doc for all other commands.